assets/scripts/async-submit.js

```diff
@@ -57,7 +57,4 @@
 
   passphraseInput.focus()
   submitButton.removeAttribute('disabled')
-
-  // Preload font awesome
-  try { document.fonts.load('14px FontAwesome') } catch (e) {}
```

Comment:
Hmm, because FontAwesome isn't in assets anymore so it triggers an unwanted error. Is it a mistake?